bigtable/client/client_options.h

```diff
@@ -53,6 +53,122 @@ class ClientOptions {
     return *this;
   }
 
+  /*
```

Comment:
Any reason we're using C-style comments instead of C++ style comments here and below?

Reply:
My bad. Changed to C++ style.